packages/twenty-front/src/modules/favorites/favorite-folder-picker/components/FavoriteFolderPickerFooter.tsx

```diff
@@ -27,6 +31,7 @@ export const FavoriteFolderPickerFooter = () => {
         <MenuItem
           className="add-folder"
           onClick={() => {
+            setIsNavigationDrawerExpanded(true);
```

Comment:
@martmull How about this? I missed this earlier and added it here -- not related to the issue.